src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java

```diff
@@ -28,20 +26,20 @@
 import org.jenkins.plugins.lockableresources.actions.ResourceVariableNameAction;
 
 @Extension
-public class LockRunListener extends RunListener<AbstractBuild<?, ?>> {
+public class LockRunListener extends RunListener<Run<?, ?>> {
 
 	static final String LOG_PREFIX = "[lockable-resources]";
 	static final Logger LOGGER = Logger.getLogger(LockRunListener.class
 			.getName());
 
 	@Override
-	public void onStarted(AbstractBuild<?, ?> build, TaskListener listener) {
+	public void onStarted(Run<?, ?> build, TaskListener listener) {
```

Comment:
🐛 Nothing to do for non-`AbstractBuild`s here, a check is needed to just `return` in that case.